tests/components/index.spec.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-undef */
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import LinesEllipsis from '../../src/index'
```

Comment:
NIT: You can leave out the `index` here. 

`import LinesEllipsis from '../../src'`